modules/golang/manifests/init.pp

```diff
@@ -14,9 +14,9 @@
   goenv::version { [
     '1.7.1',   # Used by alphagov/govuk_crawler_worker
     '1.12.1',  # Used by alphagov/govuk-csp-forwarder
-    '1.17.6',  # Used by alphagov/router (remove once router is running latest)
-    '1.17.7',  # Used by alphagov/router (remove once router is running latest)
-    '1.17.8',  # Used by alphagov/router
+    '1.17.8',  # Used by alphagov/router (remove once router is running latest)
+    '1.18.3',  # Used by alphagov/router
+    '1.',
```

Comment:
Do we need this `'1.',` ? 🤔 

```suggestion

```

Reply:
You're right, I don't think we do. That's odd, I was under the impression that that was there in the previous version, but clearly it shows up as me having added it. I'll remove and rebase.